src/main/java/nextstep/subway/exception/CommonExceptionHandler.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.subway.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class CommonExceptionHandler {
+    @ExceptionHandler(DuplicatedSectionException.class)
```

Comment:
`BAD_REQUEST`를 응답하는 핸들러들을 하나의 핸들러에서 처리하도록 수정해보면 좋을 것 같아요.
```suggestion
    @ExceptionHandler({FileSystemException.class, InvalidSectionException.class, InvalidDistanceException.class})
```

Reply:
앗 넵 확인 후 반영하겠습니다!